sonar-orchestrator/src/main/java/com/sonar/orchestrator/OrchestratorBuilder.java

```diff
@@ -292,12 +292,25 @@ public OrchestratorBuilder restoreProfileAtStartup(Location profileBackup) {
     return this;
   }
 
+  /**
+   * @deprecated since 3.15 used for old license mechanism. Use {@link #activateLicense()} instead.
+   */
+  @Deprecated
   public OrchestratorBuilder activateLicense(String pluginKey) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)